packages/cardano-services/src/Asset/CardanoTokenRegistry.ts

```diff
@@ -24,6 +25,14 @@ interface TokenMetadataServiceRecord {
   url?: StringValue;
 }
 
+const propertiesToChange: Record<string, string> = { description: 'desc', logo: 'icon' };
+export const parseTokenMetadataServiceRecord = (record: TokenMetadataServiceRecord) =>
```

Comment:
```suggestion
export const toCoreTokenMetadata = (record: TokenMetadataServiceRecord) =>
```
This tells us what we get from the function :-)

Reply:
d9ebbe08c should solve this